Mathlib/Algebra/Module/Basic.lean

```diff
@@ -300,6 +301,17 @@ theorem sub_smul (r s : R) (y : M) : (r - s) • y = r • y - s • y := by
   simp [add_smul, sub_eq_add_neg]
 #align sub_smul sub_smul
 
+instance : SMulCommClass ℤˣ R M where
```

Comment:
I'm pretty sure we already have these instances more generally; I'll remove them in a follow up

Reply:
Actually,I had time to push a fix